rpm-package/README.md

```diff
@@ -0,0 +1,52 @@
+# Jellyfin RPM
+
+## Build Fedora Package with docker
+
+Change into this directory `cd rpm-package`
+Run the build script `./build-fedora-rpm.sh`.
+Resulting RPM and src.rpm will be in `../../jellyfin-*.rpm`
+
+## ffmpeg
+
+The RPM package for Fedora/CentOS requires some additional repositories as ffmpeg is not in the main repositories.
+
+```shell
+# ffmpeg from RPMfusion free
+# Fedora
+$ sudo dnf install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm
+# CentOS 7
+$ sudo yum localinstall --nogpgcheck https://download1.rpmfusion.org/free/el/rpmfusion-free-release-7.noarch.rpm
+```
+
+## ISO mounting
+
+To allow Jellyfin to mount/umonut ISO files uncomment these two lines in `/etc/sudoers.d/jellyfin-sudoers`
+```
+# %jellyfin ALL=(ALL) NOPASSWD: /bin/mount
+# %jellyfin ALL=(ALL) NOPASSWD: /bin/umount
+```
+
+## Database patching
+
+You may need to install sqlite since CentOS has no `Recommends:` with `yum install sqlite`.
+To fix the paths in the emby database for a migration to jellyfin run the script:
+```shell
+/usr/share/jellyfin/update-db-paths.sh <path-to-library.db> <path-to-emby-data> <path-to-jellyfin-data>
+```
+PS: Please **backup your emby database beforehand**.
```

Comment:
this should not be a `PS`, it should be **the very first thing** in this section! :)

Reply:
Honestly, we should probably just stop trying with the whole "migrate DB" thing, since it breaks in unknown ways and just causes us a lot of headaches.

Reply:
I removed the hole DB migration part from the package since a RPM is probably not the right place for this anyway.